apps/dashboard/app/views/files/_globus.html.erb

```diff
@@ -0,0 +1,10 @@
+<button id="globus-btn" type="button" class="btn btn-primary btn-sm">
+  <a href="#" id="globus-btn-link" target="_blank">Globus</a>
+</button>
```

Comment:
```suggestion
  <a href="#" id="globus-btn-link" class="btn btn-primary btn-sm" target="_blank">Globus</a>
```
I think this should just be a link without the `button` tag because it doesn't modify anything. I think you can get rid of the CSS too.


As is - this is what I get 
![image](https://github.com/OSC/ondemand/assets/4874123/83b6c1cf-7ea9-4673-80e3-cae74832bac7)

with :hover, still a bit unreadable. 

![image](https://github.com/OSC/ondemand/assets/4874123/0c2fffbf-5f44-4daa-8c50-3fcddca164f7)